packages/cli/src/oclif/ZapierBaseCommand.js

```diff
@@ -127,6 +129,14 @@ class ZapierBaseCommand extends Command {
     }
   }
 
+  /**
+   * Print text in a list style.
+   * @param {string[][]} items
+   */
+  logList(items, { maxWidth = stdtermwidth } = {}) {
```

Comment:
I like having the `maxWidth` param here, but is there a particular reason you added it? I don't see it used anywhere.

Reply:
Yeah, it's not used and probably won't. Removed it in fc9fbeb.